src/extension/module.js

```diff
@@ -2891,7 +2891,8 @@
      * else <code>false</code>
      */
     isAuthenticated() {
-      return !!this.status?.profile;
+      const { status } = this.status;
+      return status && status !== 401;
```

Comment:
I think this is wrong. `this.status.status` is only set temporarily in the `fetchStatus()` method for the userInfo dropdown to react to it. I think the presence of `this.status.profile` which is coming from the admin API is the correct indication that the user has signed in ("is authenticated").